### DISCUSSION TITLE
Air SDK is compatible with Windows Server?

### DISCUSSION THREAD

**Discussion by lwmirkk** [airsdk/Adobe-Runtime-Support#2034] [Q&A]
Hi!

sorry for the silly question, but I couldn't find anything specific anywhere about AIR SDK compatible systems, if any x64 version of Windows works normally...

So, Windows Server works?

Thanks a lot.

**Comment by ajwfrost** [comment]
It's not one of the platforms that we'd normally test on, but there shouldn't be any reason why it doesn't... as long as it's equivalent to Windows 7 or later, it should work.

thanks
